### PR TITLE
Added NewBeeDrone Infinity AIO V2 config file

### PR DESCRIPTION
**Pull request by Romin0615**
Hello,

We'd like to add NewBeeDrone Infinity AIO V2 target configuration. We already tested it on the hardware. I'm hoping it's good enough to merge. Otherwise, let me know and I will work on it again. Thanks!

### CODE REVIEW

File: configs/default/NEBD-NBD_INFINITYAIOV2.config [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+feature RX_SERIAL
+feature OSD
+
+# serial
+serial 0 64 115200 57600 0 115200
+
+# master
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
```
Comment (DusKing1) [inline configs/default/NEBD-NBD_INFINITYAIOV2.config:82]: Default setting

**Review by DusKing1 — CHANGES_REQUESTED** [review]

File: configs/default/NEBD-NBD_INFINITYAIOV2.config (hunk 1 above)
Comment (Romin0615) [reply configs/default/NEBD-NBD_INFINITYAIOV2.config:82]: Are there still default settings here that have not been removed? We have checked but not sure which default settings still need to remove. Please let us know and we will fix them.

Comment (haslinghuis) [reply configs/default/NEBD-NBD_INFINITYAIOV2.config:82]: After fixing please also squash the commits.

Comment (mywinchester) [reply configs/default/NEBD-NBD_INFINITYAIOV2.config:83]: Already finished squash, please check if there have any problem, thanks.

**Comment by haslinghuis** [comment]
This PR is still based off master instead of a feature branch.

Please follow [these instructions](https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md) to create and publish a Unified Target configuration.

**Comment by DusKing1** [comment]
what?

**Comment by mywinchester** [comment]
Sorry, our colleague is trying to change our pull request branch. Is seen have some mistake on it. Can you cancel this merge? Very sorry.

**Comment by haslinghuis** [comment]
@Romin0615 you should revert the PR.

You should see something like: 
image

**Comment by haslinghuis** [comment]
Probably nothing got committed as there are zero changes made in the PR.

**Pull request merged**